server/jobs/job_manager.go

```diff
@@ -77,15 +77,15 @@ func (jm *JobManager) OnConfigurationChange(env mscalendar.Env) error {
 		if enabled && !active {
 			err := jm.activateJob(job)
 			if err != nil {
-				jm.env.Logger.Errorf("Error activating job", "id", job.id, "error", err.Error())
+				jm.env.Logger.Errorf("Error activating %s job. %s", job.id, err.Error())
```

Comment:
total nit, but why not `%v` and `err`? Just a little shorter...

Reply:
@levb Fixed, and did a quick traversal of the codebase to fix others